sbt-typesafe-conductr-tester/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+lazy val root = Project("plugins", file(".")).dependsOn(plugin)
+lazy val plugin = file("../").getCanonicalFile.toURI
```

Comment:
I'd usually just cd into one of `src/sbt-test/...` and test there, but no prob if you want the tester around.